src/operator/make_loss.cc

```diff
@@ -37,7 +37,7 @@ The output of this function is the gradient of loss with respect to the input da
 For example, if you are a making a cross entropy loss function. Assume ``out`` is the
 predicted output and ``label`` is the true label, then the cross entropy can be defined as::
 
-  cross_entropy = label * log(out) + (1 - label) * log(1 - out)
+  cross_entropy = -mean(label * log(out) + (1 - label) * log(1 - out))
   loss = MakeLoss(cross_entropy)
```

Comment:
For batch data, the [loss is already normalized by batch size in the backend](https://github.com/dmlc/mxnet/blob/master/src/operator/make_loss-inl.h#L94-L97), so `mean` is not necessary. I think you're right about the sign.